src/main/java/wolox/training/models/User.java

```diff
@@ -19,7 +19,7 @@
     @GeneratedValue(strategy = GenerationType.AUTO)
     private Long id;
 
-    @Column(nullable = false)
+    @Column(nullable = false, unique=true)
```

Comment:
I thought that if I'm using the `findByUserName` method to fecth users, `username` should be unique, in order to guarantee there's a unique username for each user. Otherwise, fetching a single user in that way wouldn't have had any sense.